<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>referenceCurvedlg</class>
 <widget class="QWidget" name="referenceCurvedlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="dateTime">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>130</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Ref.  Date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>195</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Ref.  Avg&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>260</y>
     <width>122</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Ref.  Index&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>325</y>
     <width>158</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Ref. Pro. Con&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>390</y>
     <width>153</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Ref. Val.  Con&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CalDate_linEdit">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>130</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="RefCurvAvg_lineEdit">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>195</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="RefInd_lineEdit">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>260</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProRCon_lineEdit">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>325</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValRCon_lineEdit">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>390</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="CalDate_change">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>130</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalDate_Tip">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RefCurvAvg_change">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>195</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RefCurvAvg_Tip">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>195</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RefInd_change">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>260</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RefInd_Tip">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProRCon_change">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>325</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProRCon_Tip">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>325</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ValRCon_change">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>390</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ValRCon_Tip">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pageDown">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>151</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next  Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>119</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>434</y>
     <width>120</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>434</y>
     <width>119</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Calibration:  ReferenceCurve&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
